player/lua/stats.lua

```diff
@@ -736,29 +741,31 @@ local function add_video(s)
     if ro and (r["w"] ~= ro["dw"] or r["h"] ~= ro["dh"]) then
         append_resolution(s, ro, "Output Resolution:", "dw", "dh")
     end
-    append_resolution(s, {w=scaled_width, h=scaled_height}, "Scaled Resolution:")
+    local scale = nil
     if not mp.get_property_native("fullscreen") then
-        append_property(s, "current-window-scale", {prefix="Window Scale:"})
+        scale = mp.get_property_native("current-window-scale")
     end
+    append_resolution(s, {w=scaled_width, h=scaled_height, s=scale}, "Scaled Resolution:")
+    append_property(s, "deinterlace", {prefix="Deinterlacing:"})
+
     append(s, r["pixelformat"], {prefix="Pixel Format:"})
     if r["hw-pixelformat"] ~= nil then
         append(s, r["hw-pixelformat"], {prefix_sep="[", nl="", indent=" ",
                 suffix="]"})
     end
+    append(s, r["colorlevels"], {prefix="Levels:", nl=""})
```

Comment:
maybe move it to the same line as the other color stuff?

Reply:
I think it fits nicely along pixel format. And the next line is pushing its limits if it comes to length, so I would avoid adding more to it.